windows/Picker/RNCPickerComponentView.cpp

```diff
@@ -0,0 +1,220 @@
+
+  // Listen for text submitted (when user presses Enter in editable mode)
+  m_textSubmittedRevoker = m_comboBox.TextSubmitted(
+      winrt::auto_revoke, [this](const auto& /*sender*/, const auto& /*args*/) {
+        if (m_updating) {
+          return;
+        }
+        EmitPickerSelectEvent();
+      });
+
+  m_island = winrt::Microsoft::UI::Xaml::XamlIsland{};
+  m_island.Content(m_comboBox);
+  islandView.Connect(m_island.ContentIsland());
+  m_islandView = winrt::make_weak(islandView);
+}
+
+void RNCPickerComponentView::EmitPickerSelectEvent() {
+  if (auto eventEmitter = this->EventEmitter()) {
+    const int32_t selectedIndex = m_comboBox.SelectedIndex();
+
+    PickerCodegen::RNCPicker_OnPickerSelect eventArgs;
+    eventArgs.itemIndex = selectedIndex;
+    
+    // Get the selected item value and text if available
+    if (selectedIndex >= 0 && selectedIndex < static_cast<int32_t>(m_items.size())) {
+      eventArgs.value = m_items[selectedIndex].value.value_or("");
+      eventArgs.text = m_items[selectedIndex].label;
+    } else {
+      // In editable mode, user may have typed custom text
+      // Get the text from the ComboBox
+      eventArgs.text = winrt::to_string(m_comboBox.Text());
+      eventArgs.value = eventArgs.text;
+    }
+    
+    eventEmitter->onPickerSelect(eventArgs);
+  }
+}
+
+void RNCPickerComponentView::UpdateProps(
+    const winrt::Microsoft::ReactNative::ComponentView& view,
+    const winrt::com_ptr<PickerCodegen::RNCPickerProps>& newProps,
+    const winrt::com_ptr<PickerCodegen::RNCPickerProps>& oldProps) noexcept {
+  BaseRNCPicker::UpdateProps(view, newProps, oldProps);
+
+  m_updating = true;
+
+  // Update enabled state only if changed
+  if (!oldProps || oldProps->enabled != newProps->enabled) {
+    m_comboBox.IsEnabled(newProps->enabled);
+  }
+
+  // Update editable state only if changed
+  if (!oldProps || oldProps->editable != newProps->editable) {
+    m_comboBox.IsEditable(newProps->editable);
+  }
+
+  // Update placeholder text only if changed
+  if (!oldProps || oldProps->placeholder != newProps->placeholder) {
+    if (newProps->placeholder.has_value()) {
+      m_comboBox.PlaceholderText(winrt::to_hstring(newProps->placeholder.value()));
+    } else {
+      m_comboBox.PlaceholderText(L"");
+    }
+  }
+
+  // Update items if changed, OR if editable state changed (since item format differs)
+  const bool editableChanged = !oldProps || oldProps->editable != newProps->editable;
+  if (editableChanged || !oldProps || oldProps->items != newProps->items) {
+    m_comboBox.Items().Clear();
+    m_items.clear();
+
+    for (const auto& item : newProps->items) {
+      // Store item data
+      m_items.push_back(item);
+
+      // For editable ComboBox, add items as strings for proper autocomplete behavior
+      // For non-editable, use ComboBoxItem for testID support
+      if (newProps->editable) {
+        m_comboBox.Items().Append(winrt::box_value(winrt::to_hstring(item.label)));
```

Comment:
```suggestion
        m_comboBox.Items()
                  .Append(winrt::box_value(winrt::to_hstring(item.label)));
```

Minor: fix formatting.